lib/livebook/runtime/evaluator/default_formatter.ex

```diff
@@ -27,7 +27,7 @@ defmodule Livebook.Runtime.Evaluator.DefaultFormatter do
 
   def format_result({:error, kind, error, stacktrace}) do
     formatted = format_error(kind, error, stacktrace)
-    {:error, formatted}
+    {:error, formatted, error_type(error)}
```

Comment:
@jonatanklosko @cristineguadelupe how hard would it be to send `error_type(error)` as a separate output? This would simplify the logic in live/output.ex, right?

Reply:
Nevermind, @cristineguadelupe already refactored the error rendering :)